packages/cookies-tooltip/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { setCrossDomainCookieClientSide } from '@lidofinance/utils'
+import {
+  COOKIE_ALLOWED_KEY,
+  COOKIE_VALUE_NO,
+  COOKIE_VALUE_YES,
+} from './constants'
+
+type windowOnfocusCallback = () => void
+
+export const allowCookies = (): void => {
+  setCrossDomainCookieClientSide(COOKIE_ALLOWED_KEY, COOKIE_VALUE_YES)
+}
+
+export const declineCookies = (): void => {
+  setCrossDomainCookieClientSide(COOKIE_ALLOWED_KEY, COOKIE_VALUE_NO)
+}
+
+export const setWindowOnfocus = (callback: windowOnfocusCallback): void => {
+  window.onfocus = () => {
```

Comment:
1. multiple calls will overwrite each other
2. you are not detaching event on unsubscribe.

use `useEffect` and `addEventListener`

Reply:
(or, if you're expecting to use this both with and without React, it should provide unsubscribe callback anyway and use addEventListener, not `window.onfocus`

Reply:
ok. i will use addEventListener and removeEventListener

Reply:
@Jabher see a new variant pls